test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/KeepAliveTimeoutTests.cs

```diff
@@ -63,7 +63,7 @@ public class KeepAliveTimeoutTests
                         "GET / HTTP/1.1",
                         "",
                         "");
-                    await Task.Delay(ShortDelay);
```

Comment:
I recommend putting `await Task.Delay(1).ConfigureAwait(false)` at the top of the `TestKeepAliveTimeout` to get everything of the xunit sync context.